packages/odyssey-storybook/src/components/odyssey-mui/DataTable/DataTable.stories.tsx

```diff
@@ -868,3 +868,37 @@ export const CustomFilterWithDefaultVariant: StoryObj<DataTableProps> = {
     );
   },
 };
+
+export const Truncation: StoryObj<DataTableProps> = {
+  render: function C() {
+    const columns = useMemo(
+      () => [
+        {
+          accessorKey: "truncated",
+          header: "Truncated",
+        },
+        {
+          accessorKey: "wrapped",
+          header: "Wrapped",
+          enableWrapping: true,
```

Comment:
We want this to be a boolean with an `is` or `has` prefix. I get that we're using Material React Table's data object, but maybe we should've been converting their prop names up to this point. Or maybe we want to use their props straight-up.

Just seems weird because it's not _our_ data. So if we upgrade the library again, it will break peoples' code. Same issue as MUI because we have no abstraction.